src/views/DisplayController.js

```diff
@@ -0,0 +1,21 @@
+import RACING_VARIABLES from "../constants/RacingVariables.js";
+import { Console } from "@woowacourse/mission-utils";
+
+class DisplayController {
+
+  async displayPlayerPositions(players, positions) {
+    for (let i = 0; i < players.length; i++) {
+      Console.print(`${players[i]} : ${RACING_VARIABLES.POSITION_MARKER.repeat(positions[i])}`);
+    }
+
+    Console.print("");
+  }
+
+  async displayWinners(players, winners) {
+    const winnerArray = winners.map(idx => players[idx]);
+
+    Console.print(`${RACING_VARIABLES.WINNER_PROMPT} : ${winnerArray.join(',')}`);
```

Comment:
```suggestion
    Console.print(`${RACING_VARIABLES.WINNER_PROMPT} : ${winnerArray.join(', ')}`);
```
이렇게 하면 최종 우승자가 `최종 우승자 : pobi,jun` 이런식으로 나올 것 같은데 `최종 우승자 : pobi, jun` 이렇게 잘 나오시나요??

Reply:
join 내부에 들어가는 값도 상수처리 하는 것도 좋을 것 같아요!!  🤔
```suggestion
    const SEPARATOR = ', '
    Console.print(`${RACING_VARIABLES.WINNER_PROMPT} : ${winnerArray.join(SEPARATOR)}`);
```

Reply:
> 이렇게 하면 최종 우승자가 `최종 우승자 : pobi,jun` 이런식으로 나올 것 같은데 `최종 우승자 : pobi, jun` 이렇게 잘 나오시나요??

언급해주셔서 감사합니다! npm test결과 성공이길래 해당 부분을 캐치하지 못했군요